easybuild/easyconfigs/u/UShER/UShER-0.4.1-gompi-2020b.eb

```diff
@@ -0,0 +1,45 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'UShER'
+version = '0.4.1'
+
+homepage = 'https://github.com/yatisht/usher'
+description = """UShER is now a package consisting of a family of programs for 
+rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+github_account = 'yatisht'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(namelower)s-%(version)s-isnan-CMake.patch']
+checksums = [
+    '61acdeaec61f2b376704b4a519bba44b7cffdebf315f8126aba0a80ac3268b0e',   # user-0.4.1.tar.gz
+    'c0a841bd29343057be8a2f1944aa6226cf4f4216bb641366b93250994c1ae937',   # usher-0.4.1-isnan-CMake.patch
+]
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+dependencies = [
+    ('tbb', '2020.3'),
+    ('protobuf', '3.14.0'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('Boost', '1.74.0'),
+    ('Python', '3.8.6'),
+    ('zlib', '1.2.11'),
+    ('Kent_tools', '411'),
```

Comment:
This doesn't depend on Kent tools at all. It only uses one specific tool in one example workflow file, which is a file that we don't even install. So there is literally not a single reference to this software anywhere.

Reply:
I beg to differ. The installation [instructions](https://github.com/yatisht/usher/blob/master/installCentOS.sh) clearly include faToVcf which is part of the the Kent_tools set. Anybody who is using `UShER` from the GitHub page thus expects this program to be around when installing UShER via EasyBuild. So for me this is an extension similarly to the modules we are installing in `R`. So from that angle I think it makes much sense to install that. This is particularly true as ''UShER'' is part of the ''pagnolin'' package which is widely used for Covid-19 PCR tests. It does not help to exclude this vital program, which is needed to convert one file format into another one. 
I agree with the part of how they included it is not the best way, but I disagree on the part regarding dependency. In order to use the full potential of the program, and in order how it is used in praxis, we should leave the `Kent_tool` in it as a dependency.

Reply:
It is only referenced in
1. A snakefile that we don't even install. https://github.com/yatisht/usher/blob/master/workflows/Snakefile#L82
2. A snakefile that requires (according to their own instructions) to use conda, which, of course won't be compatible with EB regardless.
Nothing that gets installed here has any connection to the other tools whatsoever.